service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -69,6 +66,7 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
                 "JOIN \n" +
                 "   itembank.testformitem formItem \n" +
                 "   ON formItem._fk_item = item._key \n" +
+                "   AND formItem._fk_adminsubject = adminItems._fk_adminsubject \n" +
```

Comment:
This is one of the bugs.  Items can be referenced on many forms across segments.  We weren't isolating that.